intelmq/bots/parsers/shadowserver/config.py

```diff
@@ -1280,3 +1281,28 @@ def validate_fqdn(value):
         'classification.identifier': 'opencwmp',
     }
 }
+
+accessible_vnc = {
+    'required_fields': [
+        ('time.source', 'timestamp', add_UTC_to_timestamp),
+        ('source.ip', 'ip'),
+        ('source.port', 'port'),
+    ],
+    'optional_fields': [
+        ('source.reverse_dns', 'hostname'),
+        ('source.asn', 'asn'),
+        ('source.geolocation.cc', 'geo'),
+        ('source.geolocation.region', 'region'),
+        ('source.geolocation.city', 'city'),
+        ('extra.', 'naics', invalidate_zero),
+        ('extra.', 'sic', invalidate_zero),
+        ('extra.', 'banner', validate_to_none),
+        ('extra.', 'product', validate_to_none),
+    ],
+    'constant_fields': {
+        'protocol.transport': 'tcp',
+        'protocol.application': 'vnc',
+        'classification.type': 'vulnerable service',
```

Comment:
This has been discussed, how do you want to proceed? Also, is that really blocking the merge of the code? IMHO the discussion is about "Do you send this out" and not "should we merge this parser".